ocs_ci/ocs/ocp.py

```diff
@@ -106,6 +106,34 @@ def exec_oc_cmd(self, command, out_yaml_format=True, secrets=None, **kwargs):
             return yaml.safe_load(out)
         return out
 
+    def exec_oc_debug_cmd(self, node, cmd_list):
+        """
+        Function to execute "oc debug" command on OCP node
+
+        Args:
+            node (str): Node name where the command to be executed
+            cmd_list (list): List of commands eg: ['cmd1', 'cmd2']
+
+        Returns:
+            out (str): Returns output of the executed command/commands
+
+        Raises:
+            CommandFailed: When failure in command execution
+        """
+        # Appending one empty value in list for string manipulation
+        cmd_list.append(' ')
+        cmd = " || echo 'CMD FAILED';".join(cmd_list)
```

Comment:
I had some comment about this in previous review but as you force pushed it's not visible here now. So please conversation. I see you already aligned with capital CMD FAILED ;) just see the other comments about variable

Reply:
ack, updated accordingly